.github/workflows/jira_server_ci.yml

```
@@ -0,0 +1,62 @@
+name: Jira Server CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: ${{ matrix.os }} / ${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}-latest
+    strategy:
+      matrix:
+        os: [Ubuntu]
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Should we add 3.5 here too? 
```suggestion
        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
```

---

never mind, saw your commit message that this fails